src/components/Transactions/Transactions.js

```diff
@@ -123,20 +147,19 @@ const Transactions = ({
               }
             )}
           </TransactionListStyled>
-          {!isShowMoreButtonHidden && (
+          {showToggleButton && (
             <Button
               theme="primary"
               margin="20px 0 0 auto"
               width="unset"
-              label={(isExpanded && t('Show less')) || t('Show more')}
+              label={
+                (isTransactionListExpanded && t('Show less')) || t('Show more')
```

Comment:
I think it would be better to change it to: `isTransactionListExpanded ? t('Show less')) : t('Show more')`